CLOUD-2968 Move base layer of AMQ image from jboss-openjdk to rhel7

gaohoward opened the pull request:
Update the amq63 base image to latest.
Also use content_set as perfered by cekit 2.2.x.

Signed-off-by: Howard Gao <howard.gao@gmail.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for other issues
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

goldmann reviewed (changes requested):
Look good, but you're missing the container section in osbs section so that OSBS can use defined repositories.

gaohoward commented:
@goldmann ok will change it. Thanks!

gaohoward commented:
@goldmann Hi Marek, just pushed the changes, Pls review. Thanks!
Howard

goldmann commented on image.yaml:
```diff
@@ -3,41 +3,56 @@ schema_version: 1
-    - path: jboss-a-mq-6.3.0.redhat-347.zip
-      description: "Artifact is available on Customer Portal: https://access.redhat.com/jbossnetwork/restricted/softwareDetail.html?softwareId=59151&product=jboss.amq.broker&version=6.3.0&downloadType=securityPatches"
-      md5: d26ed42769c93d90ad8fbe9d0a217105
+    - path: jboss-a-mq-6.3.0.redhat-356.zip
```
Optional: use `name` instead of `path`. This way the artifact is fetched directly from Brew and there is no need from cacher. More on the blog.

goldmann commented:
Looks good now, just added one comment, see above. This is optional for now, so approving.

howardgao committed:
CLOUD-2968 Move base layer of AMQ image from jboss-openjdk to rhel7

Update the amq63 base image to latest.
Also use content_set as perfered by cekit 2.2.x.

Signed-off-by: Howard Gao <howard.gao@gmail.com>

gaohoward commented:
@goldmann Thanks. I've updated the image.yaml (using name instead of path) could you approve it again?

goldmann reviewed (dismissed):
Great, it looks perfect now!

gaohoward commented:
close as it is outdated.
